tests/run/test_grammar.py

```diff
@@ -1869,68 +1868,53 @@ def __exit__(self, *args):
         with manager() as x, manager():
             pass
 
-        if not use_old_parser():
```

Comment:
Hmm. This file is copied from CPython's test suite and I usually try to avoid larger changes in order to simplify later updates by overwriting the file and keeping some changes. OTOH, it's definitely expensive to fire up the whole compiler for a bunch of simple snippets like this, so, well, I guess it's worth changing into plain code.

Reply:
This matches the current version in CPython's test suite anyway so I don't think updating it should be a problem: https://github.com/python/cpython/blob/16b6e14cc291865ecaf50fa57d0685da94496567/Lib/test/test_grammar.py#L1819

Reply:
Ah, ok, so they changed that as well (thus fully committing to the new parser). Then that's perfect.